products/compute/api.yaml

```diff
@@ -4867,6 +4867,8 @@ objects:
         properties:
         - !ruby/object:Api::Type::Enum
           name: 'routingMode'
+          update_verb: :PATCH
```

Comment:
I had to duplicate `update_url` here from `routingConfig` because of the logic in [`properties_by_custom_update`](https://github.com/GoogleCloudPlatform/magic-modules/blob/da7c2279331a5657de2b8d08fc67abe69d666b3d/provider/core.rb#L369) which will only add a `hasChange` block for properties that are 'updatable'.

Reply:
I'm surprised this doesn't generate bad changes in Ansible.

Do you think we're able to avoid polluting `api.yaml` with Terraform-specific information like this? Have you applied it to `terraform.yaml`?

Reply:
good point, I like it much better in the override.